etc/sysconfig/init

```diff
@@ -0,0 +1,8 @@
+# Common settings for normal and recovery init.
+
+# Amount of space to keep unallocated for refilling root or home later on.
+# Please keep this multiple of 500MB
+RESERVE_MB=0
```

Comment:
Yep was thinking about filesystem reserved space. Maybe change the name to RESERVE_LVM_MB or so to make it clear. Or update the comment above to reflect that.